osprey-mock-service.js

```diff
@@ -48,6 +48,14 @@ function createServerFromBaseUri (raml, options) {
   var app = osprey.Router()
   var path = (raml.baseUri || '').replace(/^(\w+:)?\/\/[^/]+/, '') || '/'
 
+  if (options.definition) {
+    app.use('/resources', function (req, res) {
```

Comment:
why `/resources`? the way I read #33 is that it should work with all resources defined in the RAML definition.